src/vdaf/prio2.rs

```diff
@@ -299,6 +301,95 @@ impl<'a> ParameterizedDecode<(&'a Prio2, usize)> for Share<FieldPrio2, 32> {
     }
 }
 
+/// A Prio2 output share comprised of a vector of field elements.
+#[derive(Debug, Clone)]
+pub struct Prio2OutputShare<F>(Vec<F>);
```

Comment:
`Prio2OutputShare` and `Prio2AggregateShare` seem to be identical, respectively, to `Prio3OutputShare` and `Prio3AggregateShare`. I think the existing `vdaf::OuputShare` and `vdaf::AggregateShare` would suffice for both VDAFs (with the change where the decoding parameter is `(vdaf, aggregation_param)`).